web-console/1-homepage/catalog.md

```diff
@@ -33,7 +35,16 @@
 
 ![Import](img/Import.png)
 
-- **Note:** Both Deploy Image and Import YAML are available options from inside the console as well, but when launching either from the homepage, there will include a "Add to Project" selection at the top to determine where to add the item.
+- Select from project will bring down an overlay panel with the option to select a project
+
+![Select](img/Select-02.png)
+
+- Once a project has been selected, the user can choose a builder image or template from that project to order as usual.
```

Comment:
@cshinn If a project does not have any items available for selection, would that project be hidden from the dropdown or is there another empty state or message that would show something like "this project has no images or templates, please select a different project" ?